### ISSUE TITLE
Call to undefined method TreeBuilder::getRootNode()

### ISSUE THREAD

**Issue by iq2** [codefog/contao-instagram#21]
Hi Kamil,

I run into a problem with the lastest version commited some days ago with a Contao 4.4.47 installation. The Contao Manager exits with an undefined error. Here is the stack-trace:

```
> Resolving dependencies using Composer Cloud v2.2.6

$ /usr/local/bin/ed-php-cli-73 -q /home/vz531nap/www/home/clients/_c44/web/contao-manager.phar.php composer install --prefer-dist --no-dev --no-progress --no-suggest --no-ansi --no-interaction --optimize-autoloader

ocramius/package-versions: Generating version class...
ocramius/package-versions: ...done generating version class
contao/manager-plugin: Generating plugin class...
contao/manager-plugin: ...done generating plugin class

Fatal error: Uncaught Error: Call to undefined method Symfony\Component\Config\Definition\Builder\TreeBuilder::getRootNode() in /home/vz531nap/www/home/clients/_c44/vendor/codefog/contao-instagram/src/DependencyInjection/Configuration.php:16
Stack trace:
-0 /home/vz531nap/www/home/clients/_c44/vendor/symfony/symfony/src/Symfony/Component/Config/Definition/Processor.php(50): Codefog\InstagramBundle\DependencyInjection\Configuration->getConfigTreeBuilder()
-1 /home/vz531nap/www/home/clients/_c44/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Extension/Extension.php(102): Symfony\Component\Config\Definition\Processor->processConfiguration(Object(Codefog\InstagramBundle\DependencyInjection\Configuration), Array)
-2 /home/vz531nap/www/home/clients/_c44/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ConfigurableExtension.php(35): Symfony\Component\DependencyInjection\Extension\Extension->processConfiguration(Object(Codefog\InstagramBundle\DependencyInjection\Configu in /home/vz531nap/www/home/clients/_c44/vendor/codefog/contao-instagram/src/DependencyInjection/Configuration.php on line 16
Loading composer repositories with package information
Installing dependencies from lock file
Package operations: 0 installs, 1 update, 0 removals
  - Updating codefog/contao-instagram (2.1.0 => 2.1.1): As there is no 'unzip' command installed zip files are being unpacked using the PHP zip extension.
This may cause invalid reports of corrupted archives. Besides, any UNIX permissions (e.g. executable) defined in the archives will be lost.
Installing 'unzip' may remediate them.
Loading from cache
Generating optimized autoload files
> Contao\ManagerBundle\Composer\ScriptHandler::initializeApplication
Script Contao\ManagerBundle\Composer\ScriptHandler::initializeApplication handling the post-install-cmd event terminated with an exception

In ScriptHandler.php line 113:
                                                                           
  An error occurred while executing the "contao:install-web-dir" command:  
                                                                           

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>...]


 Process terminated with exit code 1
 Result: General error

---
```

Thanks a lot,
Sascha

**Comment by qzminski** [comment]
Should be fixed in 2.1.2.

**Comment by iq2** [comment]
Thank you very much Kamil, everything works fine now.
